_data/sites.json

```
@@ -18942,6 +18942,18 @@
         ]
     },
 
+    {
+        "name": "Sourcegraph",
+        "url": "https://sourcegraph.com/docs/sourcegraph-accounts#how-to-delete-accounts-or-unlink-social-sign-ins",
+        "difficulty": "hard",
+        "notes": "Contact Sourcegraph support via email using your primary email (shown at <a href=\"https://accounts.sourcegraph.com\">accounts.sourcegraph.com</a> when signed in).",
```

```suggestion
        "notes": "Contact Sourcegraph support via email using your primary email (shown at [accounts.sourcegraph.com](https://accounts.sourcegraph.com) when signed in).",
```
We accept markdown now